client/request.go

```diff
@@ -130,7 +131,19 @@ func (cli *Client) sendClientRequest(ctx context.Context, method, path string, q
 		if len(body) == 0 {
 			return serverResp, fmt.Errorf("Error: request returned %s for API route and version %s, check if the server supports the requested API version", http.StatusText(serverResp.statusCode), req.URL)
 		}
-		return serverResp, fmt.Errorf("Error response from daemon: %s", bytes.TrimSpace(body))
+
+		var errorMessage string
+		if strings.Contains(resp.Header.Get("Content-Type"), "application/json") {
```

Comment:
The server returns `application/json; charset=utf-8`, and the default is `utf-8` so we should support just `application/json` too.

Realistically we'll only consume/produce utf-8, so perhaps this should check for the exact strings `application/json` or `application/json; charset=utf-8`.